integrationTests/vm/txsFee/multiShard/scCalls_test.go

```diff
@@ -86,7 +86,7 @@ func TestScCallExecuteOnSourceAndDstShardShouldWork(t *testing.T) {
 	// execute sc result with gas refund
 	txs := testContextDst.GetIntermediateTransactions(t)
 
-	intermediateTxs = testContextDst.GetIntermediateTransactions(t)
+	_ = testContextDst.GetIntermediateTransactions(t)
```

Comment:
please check also in this file if the second GetIntermediateTransactions call is needed.

Reply:
one get is enough